homeassistant/components/homewizard/sensor.py

```diff
@@ -168,11 +168,7 @@ def time_to_datetime(value: int | None) -> datetime | None:
         native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
         device_class=SensorDeviceClass.ENERGY,
         state_class=SensorStateClass.TOTAL_INCREASING,
-        has_fn=lambda data: (
-            # SKT/SDM230/630 provides both total and tariff 1: duplicate.
-            data.measurement.energy_import_t1_kwh is not None
-            and data.measurement.energy_export_t2_kwh is not None
-        ),
+        has_fn=lambda data: data.measurement.energy_import_t1_kwh is not None,
```

Comment:
Now handled in the lib: https://github.com/homewizard/python-homewizard-energy/blob/9dda67ab4753560732d80508e88b8db3add27a14/homewizard_energy/models.py#L100-L113